test/e2e/examples2_test.go

```diff
@@ -53,6 +55,13 @@ func (suite *ExamplesTestSuite2) TestSimplestExample() {
 }
 
 func (suite *ExamplesTestSuite2) TestWithBadgerExample() {
+	if isOpenShift(t) {
+		ocpVersion := os.Getenv("OCP_VERSION")
+		if len(ocpVersion) > 0 && strings.HasPrefix(ocpVersion, "4.1") {
```

Comment:
Can you explain why it is skipped only on OCP 4.1?

Reply:
At the moment it's only failing on 4.1 and we need to decide how much time we want to invest in investigating this, as we probably won't be supporting OCP 4.3 for much longer

Reply:
so It was passing before on 4.1 but not it is failing?

Reply:
Yes, that's correct.